src/Kernel.php

```diff
@@ -35,4 +35,24 @@ protected function configureRoutes(RoutingConfigurator $routes): void
             (require $path)($routes->withPath($path), $this);
         }
     }
+
+    public function getLogDir()
+    {
+        // When on the lambda only /tmp is writeable
+        if (getenv('DOCKER_COMPOSE') == false && getenv('LAMBDA_TASK_ROOT') !== false) {
```

Comment:
instead of `getenv('DOCKER_COMPOSE') == false`, can't we use APP_ENV == prod?

Reply:
fixed